build.sbt

```diff
@@ -7,7 +7,7 @@ organization := "com.github.mpilquist"
 name := "local-implicits"
 
 scalaVersion := "2.11.5"
-crossScalaVersions := Seq("2.10.5", "2.11.5")
+crossScalaVersions := Seq("2.11.5")
```

Comment:
It should do ... give it a try and feel free to make any additions to macro-compat that are needed (if any).